### PR TITLE
Update ruby version to 3.2.0

### PR DESCRIPTION
**Pull request by kentsanggds**
Update to ruby 3.2

- Had to monkeypatch kaminari due to an issue with how it handles `to` count.

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -71,3 +71,30 @@
...
+
+# Setup chrome headless driver
+Capybara.server = :puma, { Silent: true }
+
+Capybara.register_driver :chrome_headless do |app|
+  options = ::Selenium::WebDriver::Chrome::Options.new
+
+  options.add_argument("--headless")
+  options.add_argument("--no-sandbox")
+  options.add_argument("--disable-dev-shm-usage")
```
Comment (sengi) [inline spec/rails_helper.rb:83]: Good call! 👍

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -71,3 +71,30 @@
...
 end
+
+# Setup chrome headless driver
+Capybara.server = :puma, { Silent: true }
+
+Capybara.register_driver :chrome_headless do |app|
+  options = ::Selenium::WebDriver::Chrome::Options.new
+
+  options.add_argument("--headless")
+  options.add_argument("--no-sandbox")
```
Comment (sengi) [inline spec/rails_helper.rb:82]: Don't worry about it for this PR, but just fyi it should be possible to run in the sandbox. Might need to switch to our govuk-ruby-images though (hence definitely not worth holding up this PR).

File: Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.7.6
+FROM ruby:3.2.0
```
Comment (sengi) [inline Dockerfile:1]: Any reason not to use 3.2.2 or 3.2 (i.e. latest security patch)?

**Review by sengi — APPROVED** [review]

File: Dockerfile (hunk 3 above)
Comment (kentsanggds) [reply Dockerfile:1]: Not sure, just that I've tested it against ruby 3.2.0 and have sorted out the dependencies so don't want to mess around with it any more

Comment (sengi) [reply Dockerfile:1]: No probs 👍 Should be a drop-in replacement so hopefully a very simple update when you have a spare moment!

**Pull request merged**